src/main/resources/openapi.yaml

```diff
@@ -710,20 +827,19 @@ components:
           example: john@email.com
         password:
           type: string
-          example: 12345
+          example: '12345'
         phone:
           type: string
-          example: 12345
+          example: '12345'
         userStatus:
           type: integer
+          description: User Status
           format: int32
           example: 1
-          description: User Status
       xml:
         name: user
-      type: object
     Tag:
-      x-swagger-router-model: io.swagger.petstore.model.Tag
```

Comment:
Not sure what these extensions were doing, maybe @frantuma will know?

Reply:
needed by the inflector framework